types/relay-runtime/relay-runtime-tests.tsx

```diff
@@ -202,7 +202,9 @@ function storeUpdaterWithTypes(store: RecordSourceSelectorProxy<SendConversation
 // Source
 // ~~~~~~~~~~~~~~~~~~~~~
 
-store.publish(source);
+store.publish(source);   
+const get_store_item = store.getSource().get("someDataId");
+const get_store_item_typed = store.getSource().get<TConversation>("someDataId");
```

Comment:
Think we just want a expect type here, just to confirm the const takes on the type of that get?

Reply:
Its always returning Record<T>

Reply:
Yup, or a type annotation on the variable:

```suggestion
const get_store_item_typed: TConversation = store.getSource().get<TConversation>("someDataId");
```

Which one to use differs per situation. Sometimes I find that TS changes how it represents type aliases per version and using TS directly (with a type annotation) will always work whereas `$ExpectType` serializes it differently.